pkg/audit/manager.go

```diff
@@ -626,3 +645,31 @@ func logViolation(l logr.Logger, constraint *unstructured.Unstructured, enforcem
 		logging.ResourceName, violation.rname,
 	)
 }
+
+func emitEvent(constraint *unstructured.Unstructured, enforcementAction string, violation auditResult, eventRecorder record.EventRecorder, timestamp string, gkNamespace string) {
+	annotations := map[string]string{
```

Comment:
Can labels be used instead of annotations? Using labels helps in making simpler queries to get the events only related to a constraint or a resource?

Reply:
yea I wish but no... PTAL https://github.com/open-policy-agent/gatekeeper/pull/727#discussion_r454091721